.changeset/itchy-snails-smile.md

```diff
@@ -0,0 +1,6 @@
+---
+'@justeattakeaway/pie-cookie-banner': minor
```

Comment:
Unsure why this states `minor` when it is a breaking change

Reply:
It would have been populated when `yarn changeset` was run, but you can fix it up manually (you shouldn't have to do that every time).

Reply:
yes when I ran `yarn changeset` I was going to make it backward compatible but decided not to after some thought

Reply:
yes I believe so 👍